docs/src/cli/deploy-a-program.md

```diff
@@ -89,10 +89,10 @@ Data Length: 5216 (0x1460) bytes
 ### Redeploy a program
 
 A program can be redeployed to the same address to facilitate rapid development,
-bug fixes, or upgrades. Matching keypair files are generated once so that
-redeployments will be to the same program address.
+bug fixes, or upgrades.
 
-The command looks the same as the deployment command:
+The command looks the same as the deployment command (same keypair file that resides from file directory corresponding
+to <PROGRAM_FILEPATH>, this keypair file will be generated once and then reused):
```

Comment:
Sorry, I don't understand what this is trying to say anymore, and think the previous sentence was clearer. What was the exact problem with the previous language?

Reply:
How about we go back to the previous sentence that was there? With this change, there's no more reference to the generated keypair at all

Reply:
Pretty much, the `Matching keypair files are generated` is a bit unspecific for my taste (which is why I tried changing it),

perhaps repharase:
```
Matching keypair files are generated once so that
redeployments will be to the same program address.
```
into something like ?
```
Program signer stored in keypair file at <PROGRAM_FILEPATH> is 
generated once (when program is compiled for the very first time) and 
will be used to identify which on-chain program redeployment targets.
```

Reply:
How about we simplify and explain how the CLI behaves:

```
If a program id is not provided, the program will be deployed to the default address
at `<PROGRAM_NAME>-keypair.json`. This default keypair is generated during the first
program compilation.
```